packages/app/src/AppNav.tsx

```diff
@@ -514,6 +517,27 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
         : 'ok' // All alerts are green
       : 'none'; // No alerts are set up
 
+  const [searchesListQ, setSearchesListQ] = useState('');
+  const [dashboardsListQ, setDashboardsListQ] = useState('');
+
+  const filteredSearchesList = useMemo(() => {
+    if (searchesListQ === '') {
+      return logViews;
+    }
+    return logViews.filter(lv =>
+      lv.name.toLocaleLowerCase().includes(searchesListQ.toLocaleLowerCase()),
```

Comment:
Nit: These are probably a candidate for future refactoring into a function since they look identical except the `logViews` and `searchesListQ` parameter differences 👍